.github/scripts/check-rustsec-policy.py

```
@@ -228,6 +265,119 @@ def run_live() -> tuple[dict[str, Any], int]:
+                [
+                    "-i",
+                    f"{package}@{version}",
+                    "--depth",
+                    "1",
+                    "--prefix",
+                    "none",
+                    "--format",
+                    "{p}",
+                ]
+            )
+            tree = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)